core/capabilities/ccip/ccipsolana/commitcodec.go

```diff
@@ -117,7 +119,7 @@ func (c *CommitPluginCodecV1) Decode(ctx context.Context, bytes []byte) (cciptyp
 	}
 
 	return cciptypes.CommitPluginReport{
-		MerkleRoots: merkleRoots,
+		UnblessedMerkleRoots: merkleRoots,
```

Comment:
In Decode(), the onchain type is CommitInput, which doesn't tell us whether the merkle root is blessed or unblessed.
So we don't know which field to decode the root into.